devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -60,6 +60,9 @@ def parse(String description) {
 		if (event.name == "power") {
 			def powerValue
 			powerValue = (event.value as Integer)/10			//TODO: The divisor value needs to be set as part of configuration
+
+            if (device.getDataValue("manufacturer") == "SALUS")
```

Comment:
Rather than doing it like this, can you instead wait until this PR is merged and then extend the changes to work for your device as well? https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/11089/files

Reply:
That can help. 
It is better adding multiplier and division in the calculation, which can help the better compatibility.

Reply:
the above PR was merged, so you can rebase and adapt that solution here.